itools-packager/pom.xml

```diff
@@ -67,6 +67,34 @@
             <version>${maven.plugin.annotations.version}</version>
             <scope>provided</scope>
         </dependency>
+        <!-- Test dependencies -->
+        <dependency>
+            <groupId>org.apache.maven.plugin-testing</groupId>
+            <artifactId>maven-plugin-testing-harness</artifactId>
+            <version>3.3.0</version>
```

Comment:
Could you use a variable, defined in the main `pom.xml`, instead of using directly the version number?